tools/Environments/DevHome.Environments/DevHome.Environments.csproj

```diff
@@ -9,8 +9,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="CommunityToolkit.WinUI.Collections" Version="8.0.240109" />
+    <PackageReference Include="CommunityToolkit.WinUI.Behaviors" Version="8.1.240328-rc" />

```

Comment:
Let's update the versions in Common, rather than having older versions in Common and newer everywhere else.